src/Chapter1.hs

```diff
@@ -568,7 +576,15 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+
+isVowel :: Char -> Bool
+isVowel c
+    | c == 'a' = True
+    | c == 'e' = True
+    | c == 'i' = True
+    | c == 'o' = True
+    | c == 'u' = True
+    | not (c == 'a') = False
```

Comment:
In this cases, you can use `otherwise` to tell the compiler to return `False` on all other possible values
```suggestion
    | otherwise = False
```